catalogs/catalog-lakehouse-paimon/src/main/java/com/datastrato/gravitino/catalog/lakehouse/paimon/PaimonCatalogPropertiesMetadata.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright 2024 Datastrato Pvt Ltd.
+ * This software is licensed under the Apache License version 2.
+ */
+package com.datastrato.gravitino.catalog.lakehouse.paimon;
+
+import static com.datastrato.gravitino.connector.PropertyEntry.enumImmutablePropertyEntry;
+import static com.datastrato.gravitino.connector.PropertyEntry.stringRequiredPropertyEntry;
+
+import com.datastrato.gravitino.connector.BaseCatalogPropertiesMetadata;
+import com.datastrato.gravitino.connector.BasePropertiesMetadata;
+import com.datastrato.gravitino.connector.PropertyEntry;
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.ImmutableMap;
+import com.google.common.collect.Maps;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Implementation of {@link BasePropertiesMetadata} that represents Paimon catalog properties
+ * metadata.
+ */
+public class PaimonCatalogPropertiesMetadata extends BaseCatalogPropertiesMetadata {
+
+  public static final String METASTORE = "metastore";
```

Comment:
Iceberg catalog uses `catalog-backend` for the underlying catalog,  could you modify `METASTORE` to keep consistent with Iceberg catalog?